<?xml version="1.0" encoding="ISO-8859-15"?>
<launch>
    
    <!-- rviz args -->
    <arg name="rviz" default="false"/>
    
    <rosparam file="$(find semantic_info_gathering)/params/exploration_params.yaml" command="load"/>
    <node name="semantic_exploration" pkg="semantic_info_gathering" type="semantic_exploration_node.py" output="screen"/>

    <node name="path_navigation_node" pkg="semantic_info_gathering" type="path_navigation_node_position.py" output="screen"/>

    <node if="$(arg rviz)" pkg="rviz" type="rviz" name="rviz" args="-d $(find semantic_info_gathering)/rviz/config.rviz"/>
  
</launch>
